dcc-portal-pql/src/main/java/org/dcc/portal/pql/meta/MutationCentricTypeModel.java

```diff
@@ -139,7 +141,9 @@ private static ArrayFieldModel defineTranscript() {
         object(
             string("id", "transcriptId"),
             string("functional_impact_prediction_summary", "functionalImpact"),
-            object("consequence", string("consequence_type", "consequenceType")),
+            object("consequence",
+                string("consequence_type", "consequenceType"),
+                bool("coding", "coding")),
```

Comment:
I believe this is the cause of the issue. Since a single gene can have multiple transcripts, we are counting a single mutation multiple times since it can affect one or more transcripts. 

This may require an ETL change to fix.